unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -159,6 +162,77 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String trim() {
+    return UTF8String.fromString(toString().trim());
```

Comment:
Oh, yea, you're right, I just checked the UTF-8, all of its bytes are greater than 127 for the multiple bytes characters.